biz.aQute.launcher/src/aQute/launcher/Launcher.java

```diff
@@ -443,22 +444,30 @@ private int activate() throws Exception {
 				try {
 					Class<?> clazz = loader.loadClass((String) token);
 					BundleActivator activator = (BundleActivator) newInstance(clazz);
-					if (isImmediate(activator)) {
-						start(systemContext, result, activator);
+					if (isAfterBundleStart(activator)) {
```

Comment:
This PR needs include the associated doc changes to https://github.com/bndtools/bnd/blob/master/docs/_chapters/300-launching.md#embedded-activators.

So it seem to me that previously: immediate=true means start now, otherwise start after. That names of the new is methods do not make any sense when reading this code because it reads like if isAfter, then start immediately!

Reply:
My mistake. I named the method wrong and didn't realize. If fixed it and hope the new name makes it clearer now. I added the docu as well.